pkg/packagekit/updates.jsx

```diff
@@ -161,14 +157,7 @@ function cleanupChangelogLine(text) {
 
 // Replace cockpit-wsinstance-https@[long_id] with a shorter string
 function shortenCockpitWsInstance(list) {
-    list = [...list];
-
-    list.forEach((item, idx) => {
-        if (item.startsWith("cockpit-wsinstance-https"))
-            list[idx] = "cockpit-wsinstance-https@.";
-    });
-
-    return list;
+    return list.map(item => item.startsWith('cockpit-wsinstance-https') ? 'cockpit-wsinstance-https@.' : item);
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22224-95065ae6-20250718-143845-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)